testing/tests/DevExpress.ui.widgets/treeViewParts/searching.js

```diff
@@ -0,0 +1,425 @@
+/* global DATA, initTree */
+
+import TreeViewTestWrapper from "../../../helpers/TreeViewTestHelper.js";
+import $ from "jquery";
+const createInstance = (options) => new TreeViewTestWrapper(options);
+
+QUnit.module("searching");
+
+['dataSource', 'items'].forEach((optionName) => {
+    QUnit.test(`Search works even with loop/cycle in ${optionName} option`, function(assert) {
+        let options = { dataStructure: "plain", rootValue: 1, searchEnabled: true };
+        options[optionName] = [
+            { id: 1, text: "item1", parentId: 2, selected: false, expanded: false },
+            { id: 2, text: "item1_1", parentId: 1, selected: false, expanded: false },
+            { id: 3, text: "item2", parentId: 1, selected: false, expanded: false }];
+
+        let treeView = createInstance(options),
+            $parentItem = treeView.getElement().find('[aria-label="item1"]'),
```

Comment:
It's easier to understand variables with synchronized names: 
- text: "item1" -> $item1 instead of $parentItem
- text: "item1_1" -> $item1_1 instead of $childItem 
- text: "item2" -> $item2 instead of $anotherItem

Reply:
Fixed